geode-core/src/main/java/org/apache/geode/management/internal/configuration/validators/MemberValidator.java

```diff
@@ -62,18 +64,34 @@ public void validateCreate(CacheElement config, ConfigurationManager manager) {
       throw new EntityExistsException(
           "Member(s) " + members + " already has this element created.");
     }
+
+    // if there is no common member, we still need to verify if the new config is compatible with
+    // the existing ones.
+    for (Map.Entry<String, CacheElement> existing : existingElementsAndTheirGroups.entrySet()) {
+      manager.checkCompatibility(config, existing.getKey(), existing.getValue());
+    }
+  }
+
+  public String[] findGroupsWithThisElement(String id, ConfigurationManager manager) {
```

Comment:
It seems like this method is only here to allow callers to check the size of the map returned by `findCacheElement`.  Does it make sense to keep it around or can it be removed in favor a direct call to `findCacheElement` as done on line 52 of this file?

Reply:
the content of the returned array is also used by the the `delete` method in two other places. this method here allows for easier mocking for some unit tests also.